rero_ils/config.py

```diff
@@ -2808,14 +2808,65 @@ def _(x):
     'markdown_captions'
 ))
 
-# IMPORT
-# ====
-RERO_IMPORT_REST_ENDPOINTS = dict(
-    bnf=dict(
+# IMPORT FROM EXTERNAL SOURCE CONFIGURATION
+# =============================================================================
+#    Endpoint to load data from external repository. Each endpoint must be
+#    defined as a dict with the following keys:
+#      * key: (required) the endpoint key (used to build the API endpoint)
+#      * import_class: (required) the class used to import the external
+#                      document from this source.
+#      * import_size: (required) the max number of document returned when
+#                     searching on this source.
+#      * label: (required) the label used into the professional interface for
+#               this source. This label will be untranslated.
+#      * weight: (optional) Used to sort the sources into the professional
+#                interface. Default value is 100. Lower is the weight, higher
+#                is the priority.
+
+RERO_IMPORT_REST_ENDPOINTS = [
+    dict(
```

Comment:
Why we are not using the same syntax as the other endpoints:
```python
dict(
    loc=dict(        
        import_class='rero_ils.modules.imports.api:LoCImport',
        import_size=50,
        label='Library of Congress',
        weight=70
    )
...
```

Reply:
@rerowep The reasons is that any dict element could be outputed by the new `config` API endpoint. This output should include the `key` key, so we inserted it directly into the config structure instead of updated the dict before output.